src/NUnitEngine/nunit.engine.core.tests/Services/DriverServiceTests.cs

```diff
@@ -7,7 +7,7 @@
 using NUnit.Engine.Drivers;
 using NUnit.Engine.Extensibility;
 
-namespace NUnit.Engine.Services.Tests
+namespace NUnit.Engine.Services
```

Comment:
Not sure what your _namespace rule_ is as the namespaces in the test assembly are inconsistent. Some contain _Tests_ others don't.

Reply:
Up to now my personal rule has been NOT to use `.Tests` but I don't go out of my way to change the code so it's still inconsistent. I'd like to eventually create a simple minimalistic coding standard, but it hasn't been a priority. It's also a chore I somewhat dread because it can bring out big arguments about very small things.

Reply:
Nevertheless, we should address this at some point. :-)